api-gateway/src/main/resources/static/css/ownerdetail.css

```diff
@@ -7,6 +7,9 @@
     z-index: 50;
 }
 
+.image {
+    text-align: center;
+}
```

Comment:
What is the point of this? Changing the image layout is irrelevant to this story.

Reply:
It has been removed, was a change from another branch that was meant to be left on that branch